src/bottles/Bottle.java

```diff
@@ -74,25 +72,95 @@ void randomFillLevel(int capacity) {
         return bottle;
     }
 
-    // to improve
-    //static void bottleSettings() {
-    //    System.out.println("Bottle no " + bottle[i].getId() + ", capacity: " + bottle[i].getCapacity() + " l, Fill level: " + bottle[i].getFillLevel() + " l");
-    //}
+    static void bottleSettings(int howManyBottles, Bottle[] arrayName) {
```

Comment:
same as in `whichBottle` function.

`show/display/presentBottleSettings(Bottle[] bottleList)`